### ISSUE TITLE
Add login page 

### ISSUE THREAD

**Issue by Snehalgupta-07** [swamimalode07/SimonsaysGame#8]
I would like to add login page so that users can keep track of their scores.

**Comment by Snehalgupta-07** [comment]
@swamimalode07 please share your thoughts .

**Comment by swamimalode07** [comment]
I think to keep easy access ,the project should have the login method it has  but we can ensure that same user can login only once , can you make this feature?

**Comment by Snehalgupta-07** [comment]
Yes, I will make sure usernames are unique without any passwords, please assign it to me

**Comment by mesalm** [comment]
"Is the issue still open in the project? Am I allowed to work on it?"

**Comment by swamimalode07** [comment]
YEah sure , but can you think of some other way of login, than securing it

**Comment by mesalm** [comment]
We can consider using JWT (JSON Web Tokens) for authentication, as it provides a more secure way to manage logins. It allows us to ensure that the user's session is secure and can help prevent unauthorized access

**Comment by mesalm** [comment]
I’ve implemented JWT-based secure login, password hashing for registration, and token validation middleware in my pull request. Could you please review it? If everything looks good, I’d appreciate it if you could add the Hacktoberfest label.

**Comment by swamimalode07** [comment]
> Yes, I will make sure usernames are unique without any passwords, please assign it to me

can you show me a video of what you can do without passwords
